### DISCUSSION TITLE
I can't connect to the COM port. It appears empty despite having installed all the drivers.

### DISCUSSION THREAD

**Discussion by JuanAntonioGarciaBarquero** [arkypita/LaserGRBL#2186] [Q&A]
First of all, I want to thank everyone for the help. The issue I'm facing is that I can't connect to the COM port. It appears empty despite having installed all the drivers, and I've followed the steps in the instructions. Greetings and again, thank you.

**Comment by Tribo-2dot0** [comment]
Qual sua máquina? Verificou no Gerenciador de Dispositivos a porta correspondente? Baud Rate 115200?

**Comment by JuanAntonioGarciaBarquero** [comment]
My device is the laser engraver 4240. I have installed the software on 2 laptops. One is an HP EliteBook 830 G7, and I have also installed it on a laptop with the model LENOVO IdeaPad 3, model name 81W1. In both cases, we have configured the baud rate to 115200. Is it necessary to have the laser device connected to the laptop? I mention this because we haven't connected it. Greetings and thank you.

**Reply by Tribo-2dot0** [reply]
Sim é necessário qua a máquina esteja conectada e ligada para que a Porta COM correta apareça na lista de LaserGrbl.

**Comment by altraerapc** [comment]
En mi caso ocurre que la maquina es reconocida por el PC pero no envia nada.
Está en COM3 a 115200 Baudios, cuando muevo los ejes manualmente desde el LCD de la maquina identifica el movimiento, etc. Pero cuando doy la orden de Iniciar pasan los minutos y nunca comienza. En ocasiones el Buffer llega aestar en rojo pero sin hacer nada (el tiempo va pasando).
![cnc](https://github.com/arkypita/LaserGRBL/assets/151034651/2df201d3-16ff-44d4-9c90-e4eed3a0bf7e)
